tests/Struct/FindologicProductTest.php

```diff
@@ -494,6 +494,71 @@ function (Attribute $attribute) use ($expectedName) {
         );
     }
 
+    public function testNonFilterablePropertiesAreExportedAsPropertiesInsteadOfAttributes(): void
+    {
+        if (Utils::versionLowerThan('6.2.0')) {
+            $this->markTestSkipped('Properties can only have a filter visibility with Shopware 6.2.x and upwards');
+        }
+
+        $productEntity = $this->createTestProduct([
+            'properties' => [
+                [
+                    'id' => Uuid::randomHex(),
+                    'name' => 'some value',
+                    'group' => [
+                        'id' => Uuid::randomHex(),
+                        'name' => 'blub',
+                        'filterable' => false
+                    ],
```

Comment:
It would be good to test `'filterable' => true` as well

Reply:
That is the default. We already have many tests that test this behavior.